src/pa01/snakes_and_ladders.py

```diff
@@ -0,0 +1,141 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Daniil Efremov', 'Sigurd Grøtan'
+__email__ = 'daniil.vitalevich.efremov@nmbu.no', 'sgrotan@nmbu.no'
+
+import random as rd
+import statistics
+
+
+def throw_dice():
+    return rd.randint(1, 6)
+
+
+def check_ladder_or_snake(position):
+    critical_pos = {1: 40, 8: 10, 36: 52, 43: 62, 49: 79,
```

Comment:
You can consider making this variable a constant so that it will initialize once when you run the program instead of assign it every time a dice is rolled.